showcase/src/app/demo/sidebar/demo-sidebar.component.html

```diff
@@ -0,0 +1,5 @@
+<example-viewer [extensions]="['HTML', 'TS']" filesPath="sidebar/regular" exampleTitle="SHOWCASE.DEMO.SIDEBAR.TITLE">
+	<button color="primary" mat-raised-button mat-button (click)="openMenu()" translate>{{ 'SHOWCASE.DEMO.SIDEBAR.OPEN_MENU' | translate }}</button>
+	<button color="primary" mat-raised-button mat-button (click)="openLeftSidebar()" translate>{{ 'SHOWCASE.DEMO.SIDEBAR.OPEN_LEFT' | translate }}</button>
+	<button color="primary" mat-raised-button mat-button (click)="openRightSidebar()" translate>{{ 'SHOWCASE.DEMO.SIDEBAR.OPEN_RIGHT' | translate }}</button>
```

Comment:
The "translate" feature is used twice: as a directive and as a pipe. You can simply use the directive ;)